crt_portal/static/sass/custom/form.scss

```diff
@@ -95,6 +95,10 @@ form#report-form {
       margin-top: 1.5rem;
     }
 
+    &:last-of-type:not(:only-of-type) {
```

Comment:
I should probably throw a comment in here explaining what this does. I'll do that as part of the rest of the design audit finishes. (I'd add it as a suggestion now, but I can't actually remember precisely what it does, ha)